docs/changelog.md

```diff
@@ -283,25 +320,32 @@ description: |
   - Two small fixes for PDF standard conformance
 
 - Performance
-  - Typst's layout engine is now multi-threaded. Typical speedups are 2-3x for
-    larger documents. The multi-threading operates on page break boundaries, so
+  - Typst's layout engine is now multithreaded. Typical speedups are 2-3x for
+    larger documents. The multithreading operates on page break boundaries, so
     explicit page breaks are necessary for it to kick in.
   - Paragraph justification was optimized with a new two-pass algorithm.
     Speedups are larger for shorter paragraphs and range from 1-6x.
 
 - Command Line Interface
   - Added `--pages` option to select specific page ranges to export
+    <!-- Add accepted format for the argument.
+         If I say `1,2,5-6` for a one page document it won't give any warnings.
+         It also still prints 7 pages instead of 4.
+         It's a good idea to add a CLI section in the docs where everything can
+         be explained in great details. -->
   - Added `--package-path` and `--package-cache-path` as well as
     `TYPST_PACKAGE_PATH` and `TYPST_PACKAGE_CACHE_PATH` environment variables
     for configuring where packages are loaded from and cached in, respectively
   - Added `--ignore-system-fonts` flag to disable system fonts fully for better
     reproducibility
   - Added `--make-deps` argument for outputting the dependencies of the current
     compilation as a Makefile
+    <!-- Doesn't work/escape for `--make-deps Makefile 2024-10-05T14:20:16Z.typ`. -->
   - Added `--pretty` option to `typst query`, with the default now being to
-    minify
+    minify (only takes effect for JSON format)
```

Comment:
I don't think it's worth mentioning because it is not explicitly only for JSON. It's just that we only have JSON and YAML and minfying YAML doesn't really make much sense. If we had another format that can be minified, it would be minified.

Reply:
> If we had another format that can be minified, it would be minified.

True, that's why I'm talking about current state of affairs.

---

> it is not explicitly only for JSON. 

It is not _explicitly_, but it _is implicitly_ only for JSON (currently).

---

> It's just that we only have JSON and YAML and minfying YAML doesn't really make much sense.

That's what I'm talking about. The "only takes effect for JSON format" describes exactly that. Perhaps someone would think that YAML also is being minified, but no, it is in fact not being minified/compressed, but in some situations it _can_ be. So I still think that it's worth mentioning to clear any potential misunderstandings/misinterpretations.